src/cli.ts

```diff
@@ -96,6 +96,10 @@ cli
       process.env.NO_SANDBOX = '1'
     }
 
+    if (options['--']) {
+      process.env.ELECTRON_CLI_ARGS = JSON.stringify(options['--']);
```

Comment:
FWIW, when I run `yarn build` on this branch, prettier wants these semicolons removed

```
% yarn build      
yarn run v1.22.19
$ npm run lint && node scripts/build.js

> electron-vite@1.0.29 lint
> eslint --ext .ts src/**


/Users/pzich/Projects/config/razorcrest/ui/node_modules/electron-vite/src/cli.ts
  100:68  error  Delete `;`  prettier/prettier
  158:68  error  Delete `;`  prettier/prettier

✖ 2 problems (2 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.
  ```

Reply:
@pzcfg Thanks for your review, I got it.